cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -3645,16 +3684,63 @@ class RaysView extends PolyShapeView {
         }
     }
 
+    setupDragEvents(events) {
```

Comment:
why do you need two `setupDragEvents` methods?

Reply:
Because `ShapeView.setupDragEvents()` assumes that the shape consist of single SVG element, but rays shape is a group of several lines, that's why I had to override that method. Moreover, `svg.draggable.js` has the bug - when group element is dragged, it flickers, so I had to implement a workaround here.